README.MD

```diff
@@ -50,19 +55,78 @@ out of the box configuration for both `config/application-example.yml` and
 `config/database-example.yml`. Small changes that are described below are
 needed for certificates and setup of EPP/REST API.
 
+### Build docker images
+
+In the project root, run the following command:
+
+```bash
+$ docker-compose build
+```
+
+__NB!__ Containers need to be rebuilt every time there is an update to the
+gemfile.
+
+### Fire up docker-compose project
+
+Since multiple configuration files are already using container-based paths, we need to boot up our project beforehand to address these paths.
+
+In the project root, run the following command:
+
+```bash
+$ docker-compose up -d
+```
+
+This command will start all containers defined in docker-compose configuration file. You will see some errors from docker logs, but don't worry, they're expected.
+
+### Connect to active registry container
+
+Since openssl.cnf is based on paths used in container itself, we need to generate certificates inside container environment.
+
+In the project root, run the following command:
+
+```bash
+$ docker ps
+```
+
+Mark down any `CONTAINER ID` which is running `registry` image.
```

Comment:
Whats the point of this ?

Reply:
On the second thought it could be replaced with `docker run` I guess, but main intention was to get into registry env to (re)generate certificates.